yarn.lock

```diff
@@ -0,0 +1,3816 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
This project uses npm and package-lock.json. would you either delete this file or migrate the whole scripts to yarn?